ruby/src/api.cc

```diff
@@ -238,7 +238,7 @@ namespace leatherman { namespace ruby {
     {
         v = rb_funcall(v, rb_intern("to_s"), 0);
         v = rb_str_encode(v, utf8_value("UTF-8"), 0, _nil);
-        size_t size = static_cast<size_t>(rb_num2ulong(rb_funcall(v, rb_intern("bytesize"), 0)));
+        size_t size = static_cast<size_t>(rb_num2ull(rb_funcall(v, rb_intern("bytesize"), 0)));
```

Comment:
since `size_t` and `uintptr_t` are the same everywhere we care about (windows, linux, OSX, and AIX) - would it make since for us to just define our rb_num2ull symbol as returning a size_t and avoid this cast?